### PR TITLE
Improve readme fix #347

### LINKED ISSUE #347: Improve Postgresql configuration
As I said on the IRC, I'm always struggling when it comes to configure `postgresql`. If someone has a good documentation for noobs like me, I'll link it to the home page. I also think the documentation is not very pertinent regarding to my configuration (`postgresql` install via [Homebrew](http://brew.sh/) / OSX) and I'll be happy to improve it :-)

### PR DESCRIPTION
**Pull request by MartinDelille**
- add title and badge at the top
- more generic postgres setup
- minor format

### CODE REVIEW

**Comment by MartinDelille** [comment]
@Changaco ?

**Comment by Changaco** [comment]
This PR doesn't make the changes I suggested in https://github.com/liberapay/liberapay.com/issues/347#issuecomment-223906386.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
+# Liberapay
+
+[![Build Status](https://travis-ci.org/liberapay/liberapay.com.svg?branch=master)](https://travis-ci.org/liberapay/liberapay.com)
+[![Weblate](https://hosted.weblate.org/widgets/liberapay/-/shields-badge.svg)](https://hosted.weblate.org/engage/liberapay/?utm_source=widget)
+[![Donate](https://liberapay.com/assets/widgets/donate.svg)](https://liberapay.com/liberapay/donate)
+
 [Liberapay](http://liberapay.com) is a recurrent donations platform.
 
-It's a fork of Gittip/Gratipay, see [this post](https://github.com/liberapay/salon/issues/8) for the differences between the two.
+It's a fork of *Gittip/Gratipay*, see [this post](https://github.com/liberapay/salon/issues/8) for the differences between the two.
```
Comment (Changaco) [inline README.md]: What is the reason for this change?

Comment (MartinDelille) [inline README.md]: It's only a minor typo/format thing: I think all product/brand looks better when emphased especially when they use common language word like *Swift* (which is not the case here).

**Comment by MartinDelille** [comment]
@Changaco I just add the links

File: README.md (hunk 1 above)
Comment (Changaco) [inline README.md]: Please don't sneak inconsistent and controversial style changes in unrelated PRs.

**Comment by MartinDelille** [comment]
@Changaco I removed the formatting

**Comment by MartinDelille** [comment]
@Changaco up?

**Comment by Changaco** [comment]
I don't have time to deal with this right now.

**Comment by Changaco** [comment]
@MartinDelille I've made some changes, please merge if they look okay to you.
